### ISSUE TITLE
Error building wgt

### ISSUE THREAD

**Issue by Daivy03** [jellyfin/jellyfin-tizen#55]
I followed all the instructions of [jellyfin-tizen build in windows](https://github.com/jellyfin/jellyfin-tizen/wiki/Build-on-Windows),
but when I try to **Build wgt** (in cmd) with: 
`tizen.bat build-web -e ".*" -e gulpfile.js -e README.md -e "node_modules/*" -e "package*.json" -e "yarn.lock"`
I get an error and a Windows Script Host window opens, saying:

```
Script: C:\jellyfin\jellyfin-tizen\tizen.js
Row: 4
Character: 5
Error: 'console' it's not defined
Code: 800A1391
Origin: Microsoft JScript run-time error
```
**Do you know how to help me?**

**Comment by dmitrylyzo** [comment]
The error message clearly says that Windows is trying to run `tizen.js`. This happens if you run command without `.bat` (just `tizen`).

**Comment by Daivy03** [comment]
> The error message clearly says that Windows is trying to run `tizen.js`. This happens if you run command without `.bat` (just `tizen`).

Even if I remove the .bat
`tizen build-web -e ".*" -e gulpfile.js -e README.md -e "node_modules/*" -e "package*.json" -e "yarn.lock"`
 and leave i**t always gives me the same error**

**Comment by dmitrylyzo** [comment]
> Even if I remove the .bat

`.bat` must be in command for Windows.

Try specifying the full path to `tizen.bat`. Mine is `c:\tizen-studio\tools\ide\bin\tizen.bat`.

**Comment by Daivy03** [comment]
> > Even if I remove the .bat
> 
> `.bat` must be in command for Windows.
> 
> Try specifying the full path to `tizen.bat`. Mine is `c:\tizen-studio\tools\ide\bin\tizen.bat`.

**Thank You a lot. Finally I got Jellyfin.wgt**

**Comment by robotictacos** [comment]
Thanks for this.  You may want to update the Windows wiki entry, it's missing this information.

**Comment by dmitrylyzo** [comment]
> Thanks for this. You may want to update the Windows wiki entry, it's missing this information.

It is there from the beginning:
> All tools (yarn, node, tizen) are assumed to be added to the `PATH` environment variable.

and 
> tizen.bat build-web -e ".*" -e gulpfile.js -e README.md -e "node_modules/*" -e "package*.json" -e "yarn.lock"
tizen.bat package -t wgt -o . -- .buildResult

**Comment by robotictacos** [comment]
I admit that I missed the first line regarding the PATH variable.  I'm surprised that it isn't automatically added to path when you install Tizen studio.

I did see the second line but I had no idea where tizen.bat was located.  I actually thought it was supposed to be inside the jellyfin-tizen folder, so was confused there.  So personally, I was stuck until I found this issue.  I appreciate the work on the project, and the original issue and answer contained here, those enabled me to get the app running on my TV.
